### ISSUE TITLE
Start prototyping ML model for RQ4

### ISSUE THREAD

**Issue by PedroPianna** [zetenyLevay/project2-1_group15_machine_learning#93]
#### Given a target final cumulative reward and specific hardware, can a model infer the necessary hyperparameters (including training steps) to achieve that performance within a specified confidence interval?
